src/hostnet/sig.ml

```diff
@@ -4,10 +4,12 @@ module type FLOW_CLIENT = sig
   type address
 
   val connect: ?read_buffer_size:int -> address
-    -> [ `Ok of flow | `Error of [ `Msg of string ] ] Lwt.t
+    -> flow Error.t
   (** [connect address] creates a connection to [address] and returns
       he connected flow. *)
 
+  val getclientname: flow -> address
+  (** Query the address the client is bound to *)
```

Comment:
What's the intended use of this?  It's stubbed out or generates runtime errors in many places, and `grep` didn't find any internal callers, so I wonder whether it's really needed.